councilmatic_core/views.py

```diff
@@ -99,8 +99,6 @@ class IndexView(TemplateView):
     event_model = Event
 
     def get_context_data(self, **kwargs):
-        context = super(IndexView, self).get_context_data(**kwargs)
```

Comment:
Hm... yeah, it seemed odd remove, so i flagged it in it's own commit: F841
http://flake8.pycqa.org/en/latest/warnings.html?highlight=F841

> local variable name is assigned to but never used

All the other context variables were returned, and this one wasn't. Should it have been?

Seems the context object isn't actually used or changed in the `IndexView`, though perhaps I'm missing something. Perhaps the returned object in that view should be merged with that context var?